pdc/apps/common/constants.py

```diff
@@ -21,3 +21,9 @@
 
 # PDC warning field in response header
 PDC_WARNING_HEADER_NAME = 'pdc-warning'
+
+# Warning when using put method on optional parameter
+PUT_OPTIONAL_PARAM_WARNING = {
+    'PUT_OPTIONAL_PARAM_WARNING': 'When using the `PUT` method, if an optional field is not specified' +
+                                  ' in the input, it will be erased.',
```

Comment:
Do you think original text "it will be erased" is accurate? For me, describe it as something like change back to default value maybe easier for me to understand. But I am not sure, maybe original text is more precise.